usaspending_api/api_contracts/contracts/v2/download/count.md

```diff
@@ -95,6 +100,13 @@ Returns the number of transactions that would be included in a download request
 + `tas_codes` (optional, array[TASCodeObject], fixed-type)
 + `treasury_account_components` (optional, array[TreasuryAccountComponentsObject], fixed-type)
 
+## SpendingLevelObject (object)
```

Comment:
This feels like a nit-pick but I would move this up into the `Attributes` entirely. By referring to this as an `object` you are saying that the request should be:

```json
{
   "filters": { },
   "spending_level": {
      "spending_level": "transactions"
   } 
}
```